### DISCUSSION TITLE
Effects problem

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#24471] [Q&A] [answered]
This is beatmap: https://osu.ppy.sh/beatmapsets/234935#osu/545304
The problem of the effect being pinned continuously in Lazer leads to glare like the video below, compared to the stable version

https://github.com/ppy/osu/assets/128424762/9d80db37-4c03-440a-985d-0a354143a846

The dimming here is set to 65% but it's still quite blinding, I don't know if other beatmaps with a similar effect suffer.

**Comment by huyenden** [comment]
Even though I changed the renderer, the result is the same :v

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24480
